tests/modules/nf-core/quilt/quilt/nextflow.config

```diff
@@ -0,0 +1,18 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    cpus = 1 // More than 1 cpu may lead to different md5sum
```

Comment:
That is interesting, just because things end up in a different order in the output?